pilot/pkg/proxy/envoy/v2/discovery.go

```diff
@@ -86,12 +84,6 @@ type DiscoveryServer struct {
 	// APIs and service registry info
 	ConfigGenerator core.ConfigGenerator
 
-	// ConfigController provides readiness info (if initial sync is complete)
-	ConfigController model.ConfigStoreCache
-
-	// KubeController provides readiness info (if initial sync is complete)
-	KubeController *controller.Controller
-
```

Comment:
It's nice to see these controllers are decoupled from the discovery. As for the SSEDiscovery I'm currently working on implementing caching in the discovery which requires controller as a dependency. Once I implement the caching I will look into the same pattern to decouple controller.